jpeg_benchmark.py

```diff
@@ -0,0 +1,175 @@
+import gc
+import io
+import json
+import os
+import pathlib
+import time
+import datetime
+from PIL import Image
+
+# Setting parameters
+FRAME_COUNT = 300
+REAL_FPS = 30
+FRAME_TO_SAVE = 69
+INPUT_PATH = pathlib.Path("test_images", "Encode Test Dataset 2024")
+OUTPUT_PATH = pathlib.Path(f"log_{int(time.time())}")
+ENCODING = "jpeg"
+NUM_REPEATS = 1
```

Comment:
Also don't need this anymore, passing through all the different images is good enough. It takes too long anyways.